doc/content/guides/connecting-gateways/multitechconduit/_index.md

```diff
@@ -0,0 +1,330 @@
+---
+title: "MultiTech Conduit AEP"
+description: ""
+weight: 8
+---
+
+## Connecting MultiTech Conduit to {{%tts%}}
+
+The [MultiConnect® Conduit™](http://www.multitech.net/developer/products/multiconnect-conduit-platform/) is a configurable, scalable cellular communications gateway for industrial IoT applications. The technical specifications of this conduit can be found in the **[official documentation](https://www.multitech.com/documents/publications/software-guides/s000727%20-%20mPower%20Edge%20Intelligence%20Conduit_AEP_software_guide.pdf).**
```

Comment:
The link to official documentation does not need to be bold (so remove the `**` around it).

Reply:
Done.